lib/BackbeatConsumer.js

```diff
@@ -187,6 +188,9 @@ class BackbeatConsumer extends EventEmitter {
         if (this._fetchMaxBytes !== undefined) {
             consumerParams['fetch.message.max.bytes'] = this._fetchMaxBytes;
         }
+        if (process.env.RDKAFKA_DEBUG_LOGS) {
+            consumerParams.debug = process.env.RDKAFKA_DEBUG_LOGS;
```

Comment:
Is this attribute used as a boolean? In which case, might be cleaner to do:

```
consumerParams.debug = process.env.RDKAFKA_DEBUG_LOGS ? true : false;
```

Reply:
no, it is a string containing the list of rdkafka "modules" to enable logs for: `consumer`, `producer`, ... (or `all` to get everyting)